images/kubernetes-event-exporter/tests/logs.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+#
+# Tests Keycloak can be started in production mode. This requires a keystore to
```

Comment:
Couple of 'keycloak' references and production mode references in the code comments :)